### PR TITLE
Bumped Kotlin version to 1.6.10

### PR DESCRIPTION
**Pull request by cikpis**

### CODE REVIEW

File: firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/ReferenceSerializer.kt [hunk 1]
```diff
@@ -36,7 +36,6 @@ abstract class AbstractFirebaseReferenceSerializer<T> : KSerializer<T> {
                 objectEncoder.encodeObject(descriptor, 0, FieldValue.delete)
             null -> {}
         }
-        objectEncoder.endStructure(descriptor)
```
Comment (vpodlesnyak) [inline firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/ReferenceSerializer.kt:39]: I guess it was removed by mistake

File: kotlin-js-store/yarn.lock [hunk 2]
```diff
@@ -0,0 +1,3513 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (vpodlesnyak) [inline kotlin-js-store/yarn.lock]: should this file be included? seems like it's generated during the build process..

**Review by vpodlesnyak — CHANGES_REQUESTED** [review]
please correct a typo in AbstractFirebaseReferenceSerializer

File: kotlin-js-store/yarn.lock (hunk 2 above)
Comment (cikpis) [reply kotlin-js-store/yarn.lock]: yep

File: firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/ReferenceSerializer.kt (hunk 1 above)
Comment (cikpis) [reply firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/ReferenceSerializer.kt]: done

**Review by vpodlesnyak — APPROVED** [review]

**Pull request merged**